frontend/app/src/components/AssetBalances.vue

```diff
@@ -67,6 +68,12 @@ const tableHeaders = computed<DataTableHeader[]>(() => [
 ]);
 
 const sortItems = getSortItems(asset => get(assetInfo(asset)));
+
+const isExpanded = (asset: string) => some(get(expanded), { asset });
```

Comment:
@tewshi do you think we could achieve the same result with [useArraySome](https://vueuse.org/shared/useArraySome/) from vueuse?